sympy/plotting/plot_implicit.py

```diff
@@ -200,7 +200,8 @@ def _get_meshes_grid(self):
 
 
 @doctest_depends_on(modules=('matplotlib',))
-def plot_implicit(expr, x_var=None, y_var=None, **kwargs):
+def plot_implicit(expr, x_var=None, y_var=None, adaptive=True, depth=0,
+                  points=300, line_color="Blue", show=True, **kwargs):
```

Comment:
I don't know if it matters, but `blue` was changed from lowercase to uppercase here.

Reply:
Better not risk anything. I'll change it.